warp10/src/main/java/io/warp10/script/mapper/MapperCeil.java

```diff
@@ -38,6 +39,10 @@ public Object apply(Object[] args) throws WarpScriptException {
     long[] elevations = (long[]) args[5];
     Object[] values = (Object[]) args[6];
 
+    if (0 == values.length) {
+      return new Object[] {0L, GeoTimeSerie.NO_LOCATION, GeoTimeSerie.NO_ELEVATION, null};
```

Comment:
Change 0L to tick. Or else value at epoch 0 could get deleted unintentionally if overwrite flag is true.